### DISCUSSION TITLE
Using Laravel Pulse on Laravel API project

### DISCUSSION THREAD

**Discussion by bhaidar** [laravel/framework#49283] [General] [answered]
Hi team,

I wonder if I could use Laravel Pulse on a pure API project. 

I tried installing it and got this error right away when I opened it:
`Call to undefined method Illuminate\Support\Benchmark::value()`

Something else I noticed: I might need to install Livewire on this API project.

Maybe you have more info on this topic :-)

Thank you
Bilal

**Comment by NathanaelGT** [comment, marked as answer]
Make sure to update your `laravel/framework` version

`Illuminate\Support\Benchmark::value()` was released on [v10.20.0](https://github.com/laravel/framework/releases/tag/v10.20.0) (#48077)

**Reply by bhaidar** [reply]
That was so helpful @NathanaelGT! Thank you

**Comment by Sophist-UK** [comment]
> Something else I noticed: I might need to install Livewire on this API project.

Intriguing - can you explain why?
